common/scripts/kind_provisioner.sh

```diff
@@ -183,13 +183,22 @@ EOF
   fi
 
   # Create KinD cluster
-  if ! (kind create cluster --name="${NAME}" --config "${CONFIG}" -v4 --retain --image "${IMAGE}" --wait=180s); then
+  if ! (kind create cluster --name="${NAME}" --config "${CONFIG}" -v4 --retain --image "${IMAGE}" ); then
```

Comment:
Why remove wait? Does it not work if there is no CNI?

We need some way to wait

Reply:
Yeah... This isn't optimal but without a CNI this never succeeds. It'll need to be refined so that if we're not using the default CNI we install the CNI and then wait for ready.

Reply:
I think we should be fine waiting on the actual install to succeed versus waiting here - it amounts to the same thing for our purposes.